.travis.yml

```diff
@@ -8,6 +8,9 @@ matrix:
     before_install:
       - gem update --system
       - gem install bundler
+    script:
+      - bundle exec rake
+      - bundle exec rake rubocop
```

Comment:
Have you tested that this breaks when you introduce a new rubocop violation?

Reply:
Yup. Here's an example: https://travis-ci.org/Gusto/apollo-federation-ruby/jobs/551943556